packages/sw-cacheable-response/test/sw/cacheable-response.js

```diff
@@ -0,0 +1,108 @@
+importScripts(
+  '/node_modules/mocha/mocha.js',
+  '/node_modules/chai/chai.js',
+  '/node_modules/sw-testing-helpers/build/browser/mocha-utils.js',
+  '/packages/sw-cacheable-response/build/sw-cacheable-response.js'
+);
+
+const expect = self.chai.expect;
+mocha.setup({
+  ui: 'bdd',
+  reporter: null,
+});
+
+describe('Test of the CacheableResponse class', function() {
+  const validStatus = 418;
```

Comment:
I'd prefer to leave these names as-is, since the `const` does make it clear that they don't change, and they're not in the top-level scope.

(Okay, the real reason that I'd like to leave them as-is is because it makes it easier to use them later on via ES2015 object literal shorthand, e.g. `new Response('', {headers});` vs. `new Response('', {headers: HEADERS});`.)

Let me know if that's unreasonable and I'll switch 😈